src/rideAxis/index.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { space } from '../_utils/branding'
+import { ArrowIcon } from '../icon'
+
+const StyledRideAxis = styled.span`
+  display: flex;
+  align-items: center;
+  font-size: 1em;
```

Comment:
Is this really needed? I think omitting `font-size` (and `color`) will do the same.